### PR TITLE
UI for symbol search implemented

### PR DESCRIPTION
**Pull request by arunjose1995**
2019-07-05-154003_1366x768_scrot
2019-07-05-153950_1366x768_scrot
2019-07-05-153917_1366x768_scrot

### CODE REVIEW

File: landingpage/src/App.vue [hunk 1]
```diff
@@ -297,7 +350,14 @@ export default {
       searching: false,
       codefilterData: [],
       docfilterData: [],
-      tabModel: 'docsTab'
+      symbolfilterData: [],
+      tabModel: 'docsTab',
+      symbolSearch: {
+        filterByUsage: '',
+        filterByType: '',
+        usageOptions: ['all', 'use', 'define'],
```
Comment (pfitzseb) [inline landingpage/src/App.vue]: Let's use nouns here: `['all', 'usage', 'definition']`.

File: landingpage/src/App.vue [hunk 2]
```diff
@@ -191,16 +196,66 @@
...
+                    <FilterCard
+                      :search-key="searchQuery"
+                      :data="props.item"
+                    />
+                  </v-flex>
+                </v-layout>
+              </v-container>
+            </v-data-iterator>
+          </v-tab-item>
+          <!-- Code search results. Ends -->
```
Comment (pfitzseb) [inline landingpage/src/App.vue]: Should probably look into splitting all of this into a different file, but that's not critical for now.

**Comment by aviks** [comment]
Looks OK to me. I'm still not sure about the best way to display search results, but this looks like a reasonable local minimum, so let's get this merged once sebastian's comments are addressed.

File: landingpage/src/App.vue (hunk 1 above)
Comment (arunjose1995) [reply landingpage/src/App.vue]: These changes are available in the latest commit.

File: landingpage/src/App.vue (hunk 2 above)
Comment (arunjose1995) [reply landingpage/src/App.vue]: Yes. I will create a new PR for this, once it is done.

**Pull request merged**